pkgs/by-name/ya/yanic/package.nix

```
@@ -0,0 +1,25 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "yanic";
+  version = "1.5.0";
+
+  src = fetchFromGitHub {
+    owner = "FreifunkBremen";
+    repo = "yanic";
+    rev = "v${version}";
+    sha256 = "sha256-++tF6nTFIDeqWSqlBDXw/zEfl5YYa4ufzVZaUG2c8Zo=";
+  };
+
+  vendorHash = "sha256-D9V53/+C/+iv1U4kVrYWzJ8iD0MA1QcR8f5ifejFhLo=";
+
+  meta = with lib; {
+    description = "A Tool to collect and aggregate respondd data";
```

I think it's better to add the full form of the package as they advertise it in their GH description.
```suggestion
    description = "Yet another node info collector - A tool to collect and aggregate respondd data";
```

---

funny, i didn't even realize that i matched an existing phase existing this closely

---

I'm fine with changing it, just not sure about this due to:

> - meta.description must:
> - ...
> - Not start with the package name.
>   - More generally, it should not refer to the package name.

Source: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

With yanic beeing an abreviation of "Yet another node info collector" it would then kinda start with the package name.

---

Hmm, I'm aware of the rule but didn't really connect the dots since it was the full form haha.

Well, even though a technicality, it's correct. Let's leave this suggestion then.